examples/dotnet/SeleniumDocs/SeleniumManager/UsageTest.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
Please review `usings` and remove unused.

Reply:
![image](https://github.com/user-attachments/assets/2663d241-0e7a-4dd7-8d8d-5fe7b7401f86)